### ISSUE TITLE
Add newly developed models and reference results

### ISSUE THREAD

**Issue by lvanfretti** [ALSETLab/NYPAModelTransformation#2]
@marcelofcastro we need to add the new models that have been developed by you and others in the Dominion projects with reference results to the testing scripts and reference results, also generate the csv compare results.
Thanks!

**Comment by marcelofcastro** [comment]
@lvanfretti It will be done, but not in this repository. It will be done in the repository for the Dominion project, https://github.com/ALSETLab/DominionCloud-Project. I created the repo and I will help the guys in populating it :)

**Comment by lvanfretti** [comment]
@marcelofcastro ok, but that repo we cannot make public... so how are we going to integrate all the stuff related to OpenIPSL so that we can provide access as you are doing with this one?

**Comment by marcelofcastro** [comment]
Ok, so you're talking about the OpenIPSL models only, right? We can integrate them here while we test them for getting in OpenIPSL. So we have a more complete list, right?

**Comment by lvanfretti** [comment]
Yeah, but we need to decide where we are going to "centralize" everything. I think this is a good place, then you don't need to rebuild everything in another repository.

**Comment by marcelofcastro** [comment]
For sure! The dominion repository is for storing the plant models and related files, which are the models I thought you were talking about in the first moment. Here we can place the results for validation of the PSSE component models that we develop in every project.

**Comment by marcelofcastro** [comment]
Let us centralize validation here

**Comment by lvanfretti** [comment]
Alright, great!
